cdap-app-fabric/src/main/java/co/cask/cdap/internal/provision/YarnProvisioner.java

```diff
@@ -35,7 +35,7 @@
 public class YarnProvisioner implements Provisioner {
 
   public static final ProvisionerSpecification SPEC = new ProvisionerSpecification(
-    "yarn", "Default YARN Provisioner",
+    "yarn", "Default YARN",
```

Comment:
we probably shouldn't call this yarn actually. In sandbox, it doesn't run using yarn, and in the future it might run on Kubernetes or some other system. Should we rename this to NativeProvisioner?